test_collections/matter/sdk_tests/support/performance_tests/utils.py

```diff
@@ -252,12 +252,12 @@ def generate_summary(
     summary_dict["test_summary_record"]["number_of_iterations_completed"] = len(
         durations
     )
-    summary_dict["test_summary_record"][
-        "number_of_iterations_passed"
-    ] = compute_count_state(execution_status, True)
-    summary_dict["test_summary_record"][
-        "number_of_iterations_failed"
-    ] = compute_count_state(execution_status, False)
+    summary_dict["test_summary_record"]["number_of_iterations_passed"] = (
+        compute_count_state(execution_status, True)
+    )
+    summary_dict["test_summary_record"]["number_of_iterations_failed"] = (
+        compute_count_state(execution_status, False)
+    )
     summary_dict["test_summary_record"]["platform"] = "rpi"
```

Comment:
```suggestion
    summary_dict["test_summary_record"][
        "number_of_iterations_passed"
    ] = compute_count_state(execution_status, True)
    summary_dict["test_summary_record"][
        "number_of_iterations_failed"
    ] = compute_count_state(execution_status, False)
```
For some reason the `black`, changed this code formatting.

Reply:
@rquidute , I was getting additional CI failures due to black, so I locally ran the reformat... I think the repo is in a bad state. What should we do?